### ISSUE TITLE
Exec. tests in parallel (test exec. speed)

### ISSUE THREAD

**Issue by buckyroberts** [thenewboston-developers/Bank#67]

**Comment by fonar** [comment]
--- cut ---
Mikhail: agree, one of the options is simply adding pytext-xdist package
Volodymyr: Well, it’s not THAT simple, one should also take care of databases, since tests are running in parallel, one test can interfere to another, so, there should be a separate database per test thread, not a rocket science, but should be considered 
--- cut ---

@buckyroberts @vosi 
Actually, it's that simple, as it's already done by pytest-django, and pytest-xdist required as requirement for pytest-django in case we running it in parallel.

All we need is:
(1) add pytest-xdist as requirement
(2) run pytest -n <number of processes>

**Comment by vosi** [comment]
@fonar Great, what about redis?

**Comment by fonar** [comment]
@vosi frankly, not did it before, not even thinked of it, first that comes in mind, just creating auto fixture that patch settings to use 'fakeredis' insted of real. That's quick solution, and should work. But maybe there some alternative solutions.
